pkg/validation/internal/good_practices_test.go

```diff
@@ -92,3 +93,84 @@ func Test_ValidateGoodPractices(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckBundleName(t *testing.T) {
+	type args struct {
+		bundleName string
+	}
+	tests := []struct {
+		name        string
+		args        args
+		wantError   bool
+		wantWarning bool
+		errStrings  []string
+		warnStrings []string
+	}{
+		{
+			name: "should work with valid bundle name",
```

Comment:
could we add a test case for patch releases? ie `memcached-operator.v0.9.2+alpha` just to see if it works?